src/python_interpreter.rs

```diff
@@ -497,9 +498,16 @@ impl PythonInterpreter {
         let executables = if target.is_windows() {
             find_all_windows(target, min_python_minor)?
         } else {
-            (min_python_minor..MAXIMUM_PYTHON_MINOR)
+            let mut executables: Vec<String> = (min_python_minor..MAXIMUM_PYTHON_MINOR)
                 .map(|minor| format!("python3.{}", minor))
-                .collect()
+                .collect();
+            // Also try to find PyPy for cffi and pyo3 bindings
```

Comment:
It might be less confusing for the user if maturin also finds them for `cpython` bindings and then prints a warning / skips.

Reply:
I believe rust-cpython does not support PyPy. See https://github.com/dgrunwald/rust-cpython/issues/143#issuecomment-582137522